sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
+          label: '概要',
+        },
+        {
+          type: 'doc',
+          id: 'design',
+          label: 'デザイン',
+        },
+        {
+          type: 'doc',
+          id: 'requirements',
+          label: '要件',
+        },
+        {
+          type: 'doc',
+          id: 'database-configurations',
+          label: '基盤となるデータベースの設定',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-notes',
+          label: 'リリースノート',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-support-policy',
+          label: 'リリースサポートポリシー',
+        },
+        {
+          type: 'doc',
+          id: 'roadmap',
+          label: 'ロードマップ',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'クイックスタート',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'quick-start-overview',
+          label: '概要',
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Core ライブラリを使用してトランザクションを実行してみる',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb',
+              label: 'ScalarDB Core を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb-by-using-kotlin',
+              label: 'Kotlin を使用する',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Cluster を使用してトランザクションを実行してみる',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster',
+              label: 'ScalarDB Cluster を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-sql-jdbc',
+              label: 'JDBC 経由で SQL を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-sql-spring-data-jdbc',
+              label: 'Spring Data JDBC 経由で SQL を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-cluster/getting-started-with-scalardb-cluster-graphql',
+              label: 'GraphQL を使用する',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Analytics を使用して分析クエリを実行してみる',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'scalardb-analytics-postgresql/getting-started',
+              label: 'ScalarDB Analytics with PostgreSQL を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'scalardb-samples/scalardb-analytics-spark-sample/README',
+              label: 'ScalarDB Analytics を使用する',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: '管理',
```

Comment:
```suggestion
          label: '詳細',
```

or

```suggestion
          label: '参考情報',
```

or 

```suggestion
          label: 'その他の情報',
```

The English version one is `Reference`. So, `管理` doesn't seem good. `Reference` can be directly translated into `参考情報` or something. I think `詳細` is also good. But the items contained in this category seem just "other items" to me. `その他の情報` might be also good.